pyannote/database/loader.py

```diff
@@ -187,6 +187,47 @@ def __call__(self, file: ProtocolFile) -> Timeline:
         return self.loaded_[uri]
 
 
+class LabLoader:
+    """LAB loader
+
+    Parameters
+    ----------
+    path : str
+        Path to LAB file with ProtocolFile
+        (e.g. "/path/to/{uri}.lab")
+
+        each .lab file contains the segments for a single audio file, in the following format:
+        start end label
+
+        ex.
+        0.0 12.3456 sing
+        12.3456 15.0 nosing
+        ...
+    """
+
+    def __init__(self, path: Text = None):
+        super().__init__()
+
+        self.path = str(path)
+        self.data_ = load_lab(self.path)
+        
+    def __call__(self, file: ProtocolFile) -> Annotation:
+        """
+        excluded_labels : array 
+            this param can be setted in get_protocol preprocessing to avoid to add all the labels to the annotations
+            this can be useful when the task to solve is VoiceActivityDetection
```

Comment:
Can you please clarify the use case for this option?

Reply:
mmm... maybe is too specific for a general function like this one. I removed it and I will use some manual post-processing to address my specific task using functions from pyannote-core (Annotation.py)